code/modules/mob/living/carbon/carbon.dm

```diff
@@ -200,7 +200,10 @@
 					num_injuries++
 
 			if(num_injuries == 0)
-				to_chat(src, "My legs are OK.")
+				if(hallucinating())
+					to_chat(src, "<span class = 'orange'>My legs are [pick("okay","OK")].</span>")
```

Comment:
Don't mean to sound picky, but the thing it's referencing is always OK, never okay